karel.lpr

```diff
@@ -189,12 +190,12 @@ procedure hazDumpResultados(const archivo : string);
            xml.XmlFormat:=xfReadable;
            xml.Root.Name:='resultados';
 
-           if dumpMundo then begin
+           if (dumpMundo) or (dumpUniverso) then begin
               nodo:=xml.Root.NodeNew('mundos');
 
               for i:=0 to _listaMundos.Count - 1 do begin
                   mundo:=TKMundo(_listaMundos.Objects[i]);
-                  mundo.escribeAXML(nodo,true,_listaMundos[i],true);
+                  mundo.escribeAXML(nodo,true,_listaMundos[i], not dumpUniverso);
```

Comment:
no, si lo hace, la funcion escribeaXML ya tenía logica para poder dumpear todo el cuarto parametro es usa dumpselectivo y si viene en false dumpeatodo
`if not(usaDumpSelectivo) or (dumpSelectivo and _arrayDumpSelectivo[i,j]) then begin`